### DISCUSSION TITLE
tune.rfsrc - what error is being used?

### DISCUSSION THREAD

**Discussion by mcberko** [kogalur/randomForestSRC#421] [General]
Hi there,

I'm currently exploring the "tune.rfsrc" function to find the optimal mtry and nodesize for some simulation work on random survival forests. Reading the documentation, it uses "out-of-sample error". I assume this is the usual 1 - C-index that is available from survival forest objects?

Thank you!

**Comment by ishwaran** [comment]
Yes that is correct.  The error rate reported for survival is 1-C.

In general the error rate reported is the default error rate set by the package.   The following table might be useful to you if you use the function for other families:  https://www.randomforestsrc.org/articles/getstarted.html#prediction-error
